src/util/Util.js

```diff
@@ -99,11 +99,16 @@ Util.transformErrorXHR = function(xhr) {
   }
   // Replace error messages
   if (!_.isEmpty(xhr.responseJSON)) {
-    const errorMsg = xhr.responseJSON.errorCode
+    const { errorCode } = xhr.responseJSON;
+    const untranslatedErrorCodes = [
```

Comment:
nit: name it something like `noTranslationNeededErrorCodes`

Reply:
It's a pretty locally scoped tmp var, I think it's ok as-is?

Reply:
yeah, its alright. Just a nit comment